ddtrace/contrib/datadog_lambda/__init__.py

```diff
@@ -0,0 +1,58 @@
+"""
+The datadog_lambda integration currently enables traces to be sent
+before an impending timeout in an AWS Lambda function instrumented with the
+`Datadog Lambda Python <https://github.com/DataDog/datadog-lambda-python>`_ package.
+
+Enabling
+~~~~~~~~
+
+The datadog_lambda integration is enabled automatically for AWS Lambda
+functions which meet one of the following conditions::
+
+    1. The environment variable ``DD_LAMBDA_HANDLER`` is set.
+
+    When this variable is set, it's assumed that the function was automatically
+    instrumented. Therefore, a Datadog Lambda Layer (``datadog-lambda-py``), and
+    a Datadog Extension Layer (``datadog-lambda-extension``) are set.
+
+    2. The AWS Lambda function was instrumented manually by following
+    the recommended `installation <https://docs.datadoghq.com/serverless/guide/handler_wrapper/>`_ by Datadog.
+
+
+The patch aims to get the arguments provided to the ``datadog_lambda_wrapper``.
```

Comment:
do users need to know this? It kinda seems like implementation details that could change. I actually don't mind being a bit more transparent about how it works, I just want to make sure it doesn't enable users to depend on the implementation details (or that the details go stale if the approach changes)

Reply:
I actually changed my mind and I agree. This is not needed, even though we explain how we do it, it makes sense to remove this. The implementation might change and with Universal Instrumentation for Lambda, this would be removed eventually.